h2o-py/h2o/model/extensions/fairness.py

```diff
@@ -254,7 +260,9 @@ def fair_pr_plot(self, frame, protected_columns, reference, favorable_class, fig
         """
         import h2o
         from h2o.utils.typechecks import assert_is_type
-
+        from h2o.utils.shared_utils import can_use_numpy
+        if not can_use_numpy():
+            raise ImportError("numpy is required for fair_pr_plot.")
```

Comment:
Doesn't use `numpy` directly but indirectly so it's not imported in this function but it is required.